webapp/components/test-file-results.html

```diff
@@ -205,84 +125,21 @@
       }
 
       async loadResultFile(testRun) {
-        const url = this.resultsURL(testRun, this.path);
+        const url = this.resultsURL(testRun);
         const response = await window.fetch(url);
         if (!response.ok) {
           return null;
         }
         return response.json();
       }
 
-      resultsURL(testRun, path) {
+      resultsURL(testRun) {
```

Comment:
This looks like a bad merge. Part of e055778117cebff2910b93ce7c92601c73201c10 is reverted, which will break the line chart.

Reply:
Addressed, I think. Plz double-check new diff.